website_sale_credit_point/controllers/website_sale.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2016-2017 Camptocamp SA
```

Comment:
wrong copyright 2016-2022

Reply:
Thanks for your review. This PR has been updated.